packages/sommelier.rb

```diff
@@ -0,0 +1,39 @@
+require 'package'
+
+class Sommelier < Package
+  description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
+  homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
+  version '0.20'
+  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
```

Comment:
Easier to use `source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+archive/0.20.tar.gz'`?  Then you don't need the git clone, correct?

Reply:
No. The shasum is different every time. The README stays the same, because it's so old. They pack their tarballs on the fly and send them to you.

Reply:
Wouldn't they bump the version after any new changes?  This is a git repo.

Reply:
No. The README literally doesn't have anything in it but `Sommelier is a nested Wayland compositor with X11 forwarding support.`

Reply:
wouldn't it make sense to pull our own copy once a release and host it ourselves somewhere? that way we avoid shasum changing on each pull?

Reply:
It sounds like the folks at Google need a lesson about tagging new releases.

Reply:
No. If anything, I'll change the file from the README to a hashed version of `install.sh`. What do you think, @uberhacker?

Reply:
I don't think this is the correct approach.  This is a hashed version: https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20

Reply:
Yes, but it's literally a hash. Try to wget it and cat the file.

Reply:
Not sure I follow.  Any file from this tagged release should remain static.  Are you sure you aren't pulling from the master branch?

Reply:
No. Try this: `wget -q -Ook https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT; cat ok; rm ok;`

Reply:
They tag it just fine. If you create a tarball, then create another tarball with the same contents, the hashes are different. They don't save a copy of that tarball, they just make another one when it's requested from their server.

Reply:
What kind of repo is Google hosting?  This is all messed up.

Reply:
There hasn't been a release in 4 months and they probably won't. They made a newer one that has an icon on the taskbar.

Reply:
It's not like GitHub, which saves a tarball for each release. It just creates one when you ask for it.

Reply:
Saves space, but not friendly to package managers.

Reply:
It works. Why not merge it?

Reply:
I'm going to play with this a bit before I decide to merge.  I'm not convinced this is the best approach yet.

Reply:
There's a newer version from https://chromium.googlesource.com/chromiumos/platform2 that I can extract if you give me the time.

Reply:
I'm definitely up for a newer version.  Close this PR?

Reply:
I got a new version with no need for the `sed`s. The version is the first five characters of the commit ID, though.